### PR TITLE
Add rake tasks for access key management

### PR DESCRIPTION
**Pull request by mpalmer**

### CODE REVIEW

File: lib/tasks/active_stash.rake [hunk 1]
```diff
@@ -23,7 +24,7 @@ end
 namespace :active_stash do
   desc "Login to stash workspace"
   task :login, [:workspace] do |task, args|
-    CipherStash::Client::Profile.create(ENV.fetch("CS_PROFILE_NAME", "default"), ActiveStash::Logger, workspace: args[:workspace])
+    CipherStash::Client::Profile.create(ENV.fetch("CS_PROFILE_NAME", "default"), Rails.logger, workspace: args[:workspace])
```
Comment (coderdan) [inline lib/tasks/active_stash.rake:27]: Why did this change? `ActiveStash:Logger` wraps the Rails logger but allows you to override it. Though it should actually be `ActiveStash::Logger.instance`.

File: lib/tasks/active_stash.rake [hunk 2]
```diff
@@ -41,6 +42,33 @@ namespace :active_stash do
...
   end
 
+  namespace :access_key do
+    desc "Create a new access key"
+    task :create, [:name] => :environment do |_task, args|
+      if args[:name].nil?
+        error("Must provide a key name")
+        exit 1
+      end
+      puts CipherStash::Client.new(logger: Rails.logger).create_access_key(args[:name]).secret_key
```
Comment (coderdan) [inline lib/tasks/active_stash.rake:52]: Use `ActiveStash::Logger.instance`

**Review by coderdan — CHANGES_REQUESTED** [review]
Just the logger stuff but otherwise LGTM.

I like terminal table, too! I'll use that.

File: lib/tasks/active_stash.rake (hunk 1 above)
Comment (mpalmer) [reply lib/tasks/active_stash.rake:27]: The methods on `ActiveStash::Logger` don't implement the complete `Logger` interface, which StashRB takes advantage of (specifically, passing progname as the argument and rendering the message in a block).  I'll switch to using `ActiveStash::Logger.instance`, although IMO it should be `s/STDOUT/$stderr/`, and probably have a class-level `attr_accessor :logger` (or perhaps `def instance=(l); @logger = l; end`?), for full flexibility to override.

**Review by coderdan — APPROVED** [review]

**Pull request merged**